### DISCUSSION TITLE
OpenDTU vs. AhoyDTU

### DISCUSSION THREAD

**Discussion by themanfrommoon** [tbnobody/OpenDTU#716] [Q&A]
Moin,

ich habe schon eine Weile gesucht und viel gelesen, aber bisher keine Antworten auf folgende Fragen gefunden:

Warum gibt es OpenDTU "und" AhoyDTU?
Warum sind zwei Projekte nötig?
Was ist der Unterschied?
Warum lässt man nicht ein Projekt sterben und unterstützt dann das andere Projekt?
Welches Projekt ist "das Original"?
Warum hat sich ein zweites Projekt gegründet?

Beste Grüße,
Chris

**Comment by Solarteur** [comment]
1. Ausgangspunkt war Ahoy basierend auf ESP8266-Boards.
2. Dann kam das OpenDTU-Projekt basierend nur auf ESP32-Boards dazu.
3. Inzwischen gibt es bei Ahoy die Wahl zwischen ESP8266- und ESP32-Boards.
4. Mit ESP32-Boards können beide Prokekte hardwareseitig pinkombatibel eingerichtet werden, so dass jeder die Möglichkeit hat, beide Projekte zu testen und seinen Favoriten auszusuchen. 
5. Ansonsten freie Marktwirtschaft - und das ist gut so.

**Comment by themanfrommoon** [comment]
Gibt es denn Feature Unterschiede? (ausprobieren will ich sowieso beide, nur den Sinn hab ich noch nicht verstanden?!)

In der Marktwirtschaft geht es ums Geld. Da die Projekte open source sind, kann das ja schonmal nicht sein?!

**Reply by lethuer** [reply]
Ganz nette Übersicht:

https://blog.helmutkarger.de/balkonkraftwerk-teil-8-opendtu-und-ahoydtu-fuer-hoymiles-wechselrichter/

**Reply by nbduck** [reply]
Und, was nutzt Du inzwischen? Stehe vor der gleichen Frage... Gruß

**Reply by themanfrommoon** [reply]
OpenDTU .....AHOY habe ich aber gar nicht intensiv getestet. Ich wollte es parallel testen. Parallelbetrieb ist aber nicht möglich.

**Reply by stefan123t** [reply]
Der größte Unterschied besteht aktuell m.E. in der Unterstützung von MI-Wechselrichtern, das gibt es bisher nur bei AhoyDTU.
Wie es mit der Zahl der Wechselrichter aussieht weiß ich nicht, es gibt Leute die 16 Wechselrichter mit AhoyDTU abfragen/betrieben.

Ja ein Parallelbetrieb mit zwei DTUs an den selben Wechselrichtern ist nicht zu empfehlen, da sich die beiden DTUs dann um die Aufmerksamkeit des Wechselrichters auf der Funkstrecke kloppen und es zu Übertragungsfehlern kommt. Das ist aber m.E. unabhängig davon ob es Ahoy, OpenDTU oder Hoymiles DTU ist. Nur die Hoymiles DTU sendet / fragt nicht so häufig wie die anderen beiden.

Wenn man sich mal genau umsieht gibt es auch noch OpenDTU-onBattery und in Zukunft einige Projekte die die HMS-WiFi Modelle unterstützen werden. Ob die auch irgendwann noch von OpenDTU und/oder AhoyDTU unterstützt werden ist bisher noch nicht geklärt.
